### PR TITLE
Removed download_detail test from test_integration

### PR DESCRIPTION
**Pull request by audreymarthin**
Removed download_detail from data.py then made sure that all the notebooks and testing files can run without any issues.

### CODE REVIEW

**Comment by vengroff** [comment]
Looks like there is still one test trying to call `download_detail`. This is causing the checks above to fail. See https://github.com/vengroff/censusdis/actions/runs/5663896029/job/15346331660 for the error message. Should be easy to fix.

**Comment by vengroff** [comment]
This is the failing test: https://github.com/audreymarthin/censusdis/blob/main/tests/test_integration.py#L73

It had previously been there to catch the deprecation warning, but since the function is completely gone now, it can be removed as well.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (dat-sudo) [inline .vscode/settings.json:1]: looks like this file shouldn't be here

**Review by dat-sudo — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (vengroff) [reply .vscode/settings.json:1]: Yup. Probably best to add `.vscode` to `.gitignore` so this and related files are ignored going forward.

**Comment by audreymarthin** [comment]
Thank you for pointing out the test, Darren! It seems I have missed that one. And will resolve the .vscode file as well, thank you for pointing that out @dt

> This is the failing test: https://github.com/audreymarthin/censusdis/blob/main/tests/test_integration.py#L73
> 
> It had previously been there to catch the deprecation warning, but since the function is completely gone now, it can be removed as well.

Thank you for pointing out the test, Darren! It seems I have missed that one.

File: .vscode/settings.json (hunk 1 above)
Comment (audreymarthin) [reply .vscode/settings.json:1]: Yes, that is true. Thank you for pointing this out as well!

**Review by vengroff — APPROVED** [review]
Looks good and tests are passing.

**Pull request merged**